src/gui/rss/automatedrssdownloader.cpp

```diff
@@ -90,13 +87,15 @@ AutomatedRssDownloader::AutomatedRssDownloader(const QWeakPointer<Rss::Manager>
     // Update matching articles when necessary
     ok = connect(ui->lineContains, SIGNAL(textEdited(QString)), SLOT(updateMatchingArticles()));
     Q_ASSERT(ok);
-    ok = connect(ui->lineContains, SIGNAL(textEdited(QString)), SLOT(updateMustLineValidity()));
-    Q_ASSERT(ok);
     ok = connect(ui->lineNotContains, SIGNAL(textEdited(QString)), SLOT(updateMatchingArticles()));
     Q_ASSERT(ok);
-    ok = connect(ui->lineNotContains, SIGNAL(textEdited(QString)), SLOT(updateMustNotLineValidity()));
+    ok = connect(ui->lineEFilter, SIGNAL(textEdited(QString)), SLOT(updateMatchingArticles()));
```

Comment:
No, for the others we want to leave them as textEdited so that they don't rapid fire when the selection changes. updateMatchingArticles gets called explicitly at the end of the selection change handler